lib/versioned/runner.js

```diff
@@ -97,7 +98,19 @@ function installPackages(cb) {
 function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
-  spawn('node', [testFile], function testHandler(err) {
+  let args = [testFile]
+  if (process.env.packageType && process.env.packageType === 'module') {
+    // The test loader is defined in the agent, so this uses process.cwd() instead of __directory.
+    const cwd = process.cwd().slice(0, process.cwd().indexOf('/test/'))
```

Comment:
it's unfortunate we've designed a runner that is very hard to unit test. but i confirmed this works manually. but thinking more of this if we shift all this logic to the test we could test it and hten just pass the `loader` in the spawn, thoughts?

Reply:
Thinking this through...this code is only necessary here to get the directory of the agent running it. It's kind of awkward, and I agree that would be better handled by the agent. 

If we added it in the agent code as an environment variable passed to the spawn there, I'd think we could just rely on the loader path env var.

Reply:
I'm not sure I understand what you're saying.  I was just thinking of moving this logic from `lib/versioned/runner` to `lib/versioned/test` so we could test the logic.

Reply:
I was reading that as a recommendation to move this code entirely out of runner and into the node agent. I can agree with adding it to this repos tests to test the logic...though the runner is still going to need to know where to find a loader when this runner is kicking off versioned tests, so some version of pathway detection will still need to be here. (Though if we're bundling a test loader in this repo, we could use `__dirname` instead of the path manipulation here.)